content/docs/command-reference/params/diff.md

```diff
@@ -19,35 +19,28 @@ positional arguments:
 
 ## Description
 
-This command provides a quick way to compare parameter values among experiments
-in the repository history. Requires that Git is being used to version the
-project params.
+Provides a quick way to compare parameter values among experiments in the
+repository history. Requires that Git is being used to version the project
+params.
 
-> Parameter dependencies are defined with the `-p` option in `dvc run`. See also
-> `dvc params`.
+> Parameter dependencies are defined in the `params` field of `dvc.yaml` (e.g.
+> with the the `-p` (`--params`) option of `dvc run`).
 
 Without arguments, this command compares parameters currently present in the
 <abbr>workspace</abbr> (uncommitted changes) with the latest committed version.
-
-Supported parameter _value_ types are: string, integer, float, and arrays. DVC
-itself does not ascribe any specific meaning for these values.
-
-❗ By default it only shows parameters that were changed.
+This includes everything in the default parameters file (`params.yaml`) as well
+as all `params` found in [DVC files](/doc/user-guide/dvc-files). Only params
```

Comment:
"as well as all `params` found in" - this is not very clear to me

Reply:
Updated to "as well everything listed in the `params` field of `dvc.yaml`/`dvc.lock`"

p.s. had to expand "DVC files" since we're talking about a specific field, and it felt necessary keep "dvc.lock" since it's not mentioned anywhere else in the doc.